src/InteractiveWindow/Editor/InteractiveWindow.UIThreadOnly.cs

```diff
@@ -2512,7 +2512,7 @@ private DataObject Copy(NormalizedSnapshotSpanCollection spans)
             {
                 var text = GetText(spans);
                 var blocks = GetTextBlocks(spans);
-                var rtf = _rtfBuilderService.GenerateRtf(spans, TextView);
+                var rtf = ((IRtfBuilderService2)_rtfBuilderService).GenerateRtf(spans, CancellationToken.None);
```

Comment:
I don't know the rest of your code, but I'm guessing this is cancellable because it can take a while on huge selections. Do we have to be uncancellable in this context, or is there a way to make this cancellable?